package.json

```
@@ -12,7 +12,8 @@
     "lint:fix": "eslint \"./**/*.js\" --fix",
     "prettier": "prettier --check \"**/*.js\"",
     "prettier:fix": "prettier --write \"**/*.js\"",
-    "prepare":"webpack"
+    "prepare":"webpack",
+    "check": "npm run test && npm run lint && npm run prettier"
```

```suggestion
    "check": "npm run test && npm run lint && npm run prettier && npm run prepare"
```

Makes sense to run the webpack build in the check script as well